### PR TITLE
Add listOracles rpc

### PR DESCRIPTION
**Pull request by jingyi2811**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:

Add listOracles rpc

#### Which issue(s) does this PR fixes?:

Fixes part of #202

### CODE REVIEW

**Review by monstrobishi — DISMISSED** [review]

File: packages/jellyfish-api-core/src/category/oracle.ts [hunk 1]
```diff
@@ -128,6 +137,6 @@ export interface UTXO {
 export interface OracleTokenPrice {
   token: string
   currency: string
-  amount: number
-  timestamp: number
+  amount: number // example: 0.5
+  timestamp: number // example: 1623161076073
```
Comment (fuxingloh) [inline packages/jellyfish-api-core/src/category/oracle.ts:141]: Use JSDoc `@example` syntax

File: packages/jellyfish-api-core/src/category/oracle.ts [hunk 2]
```diff
@@ -84,6 +84,15 @@ export class Oracle {
...
     return await this.client.call('getoracledata', [oracleid], 'number')
   }
+
+  /**
+   * Returns array of oracle ids.
+   *
+   * @return {Promise<string[]>}
+   */
+  async listOracles (): Promise<string[]> {
+    return await this.client.call('listoracles', [], 'number')
```
Comment (fuxingloh) [inline packages/jellyfish-api-core/src/category/oracle.ts:94]: Question: there isn't pagination for this?

**Review by fuxingloh — CHANGES_REQUESTED** [review]

File: packages/jellyfish-api-core/src/category/oracle.ts [hunk 3]
```diff
@@ -128,6 +137,12 @@ export interface UTXO {
 export interface OracleTokenPrice {
   token: string
   currency: string
+  /**
+   * @example 0.5
+   */
   amount: number
+  /**
+   * @example 1623161076073
```
Comment (fuxingloh) [inline packages/jellyfish-api-core/src/category/oracle.ts:145]: @monstrobishi is this format correct?

Comment (monstrobishi) [reply packages/jellyfish-api-core/src/category/oracle.ts:145]: Should be divided by 1000

**Review by monstrobishi — CHANGES_REQUESTED** [review]

File: packages/jellyfish-api-core/src/category/oracle.ts (hunk 3 above)
Comment (fuxingloh) [reply packages/jellyfish-api-core/src/category/oracle.ts:145]: @jingyi2811 be detailed about this in JSDoc write out the specification.

**Review by fuxingloh — APPROVED** [review]

**Review by monstrobishi — APPROVED** [review]

**Pull request merged**